content/docs/reference-javascript-environment-requirements.md

```diff
@@ -1,14 +1,14 @@
 ---
 id: javascript-environment-requirements
-title: JavaScript Environment Requirements
+title: जावास्क्रिप्ट एनवायरनमेंट आवश्यकताएँ
 layout: docs
 category: Reference
 permalink: docs/javascript-environment-requirements.html
 ---
 
-React 16 depends on the collection types [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) and [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set). If you support older browsers and devices which may not yet provide these natively (e.g. IE < 11) or which have non-compliant implementations (e.g. IE 11), consider including a global polyfill in your bundled application, such as [core-js](https://github.com/zloirock/core-js) or [babel-polyfill](https://babeljs.io/docs/usage/polyfill/).
+React 16 [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) और [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) संग्रह के प्रकार पर निर्भर करता है। यदि आप पुराने ब्राउज़रों और उपकरणों का समर्थन करते हैं, जो अभी तक ये मूल रूप से प्रदान नहीं कर सकते हैं (जैसे IE <11) या जिनके पास गैर-अनुरूप कार्यान्वयन हैं (उदाहरण IE 11), तो अपने बंडल किए गए एप्लिकेशन में एक वैश्विक polyfill शामिल करें, जैसे कि [core-js](https://github.com/zloirock/core-js) या [babel-polyfill](https://babeljs.io/docs/usage/polyfill/)।
```

Comment:
संग्रह के प्रकार पर निर्भर करता है could be better as collection type in hindi to make it more understandable

Reply:
ब्राउज़रों -> browsers

Reply:
जो अभी तक ये मूल रूप से प्रदान नहीं कर सकते हैं -> जो अभी तक ye type natively (in hindi) प्रदान नहीं karte

Reply:
गैर-अनुरूप -> too complex, could be non compliant in hindi too?

Reply:
> ब्राउज़रों -> browsers

in Hindi Glosary we have a translated version of Browser
#2

Reply:
> ब्राउज़रों -> browsers

in Hindi Glosary we have a translated version of Browser
#2

Reply:
I mean ब्राउज़र्स, ब्राउज़रों is weird